docs/medical-api/handling-data/scheduled-queries.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Scheduled Queries"
+icon: "calendar"
+description: "Query the networks on your schedule."
+---
+
+# Overview
+Scheduled Queries is an automated and scheduled version of [Start Document Query](/medical-api/api-reference/document/start-document-query).
+
+It allows you to specify a schedule in which document queries should be executed automatically.
+
+You can choose one of the frequencies below for each of the document query source:
+ - Weekly: runs every Saturday.  
+ - Biweekly: runs every other Saturday, starting with the first Saturday of the month.  
+ - Monthly: runs the last weekend of the month.  
```

Comment:
Please use the actual value that users will choose on settings: lower-case. Like:
- `weekely`: runs...
- `biweekly`: runs...
- `monthly`: runs...